File: C++/URI_1013.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int a, b, c;
    cin >> a >> b >> c;
    if(a>=b && a>=c){
        cout << a << " eh o maior\n";
    }
    else if(b>=a && b>=c){
        cout << b << " eh o maior\n";
    }
    else{
        cout << c << " eh o maior\n";
    }
    
    return 0;
}

File: Python/URI_1013.py
lista = input().split(" ")
a = int(lista[0])
b = int(lista[1])
c = int(lista[2])

maiorAB = ((a + b) + abs(a - b)) / 2

if maiorAB > c:
    print(f'{int(maiorAB)} eh o maior')
else:
    print(f"{c} eh o maior")